webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -1124,12 +1091,14 @@ class StringFormatter private (
 
     override def getOntologySchema: Option[OntologySchema] = iriInfo.ontologySchema
 
-    override def checkApiV2Schema(allowedSchema: ApiV2Schema, errorFun: => Nothing): SmartIri =
-      if (hasApiV2SchemaNotEqualTo(allowedSchema)) { errorFun }
-      else { this }
+    override def isApiV2Schema(allowedSchema: ApiV2Schema): Boolean =
+      iriInfo.ontologySchema match {
+        case Some(value) => value == allowedSchema
+        case None        => true
+      }
 
-    override def hasApiV2SchemaNotEqualTo(allowedSchema: ApiV2Schema): Boolean =
-      iriInfo.ontologySchema.exists(_ != allowedSchema)
+    // override def hasApiV2SchemaNotEqualTo(allowedSchema: ApiV2Schema): Boolean =
+    //   iriInfo.ontologySchema.exists(_ != allowedSchema)
```

Comment:
Could this already be removed?

Reply:
Unless there will be no other changes requests from @seakayone in this PR, I let @BalduinLandolt to remove it in his task, which will be continuation of SF adventures.